src/ontology/cl-edit.owl

```diff
@@ -31484,6 +31487,36 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:27776108") oboInOwl:hasE
 AnnotationAssertion(rdfs:label obo:CL_4033040 "lung resident memory CD8-positive, CD103-positive, alpha-beta T cell")
 EquivalentClasses(obo:CL_4033040 ObjectIntersectionOf(obo:CL_4033039 ObjectSomeValuesFrom(obo:RO_0002104 obo:PR_000001010)))
 
+# Class: obo:CL_4033041 (alveolar macrophage CCL3-positive)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:37291214") obo:IAO_0000115 obo:CL_4033041 "An alveolar macrophage that expresses CCL3.")
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4033041 <https://orcid.org/0000-0001-6677-8489>)
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4033041 <https://orcid.org/0000-0002-9185-3994>)
+AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4033041 "2023-06-15T12:53:08Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label obo:CL_4033041 "alveolar macrophage CCL3-positive")
+SubClassOf(obo:CL_4033041 obo:CL_0000583)
+
+# Class: obo:CL_4033042 (alveolar macrophage metallothionein-positive)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:37291214") obo:IAO_0000115 obo:CL_4033042 "An alveolar macrophage that expresses metallothionein.")
```

Comment:
If there are any other distinguishing features (e.g., marker genes) or functions consider adding to definition or comment.

Reply:
Thank you for your comment. 
I have added the markers that the reference provided in Supplementary Table 6 (see screenshot):
![image](https://github.com/obophenotype/cell-ontology/assets/94959119/7ea0dcee-5f6e-4f6f-97ef-e6ea9b428ec0)
Interestingly, these markers differ from other sources (eg. Azimuth):
<img width="876" alt="image" src="https://github.com/obophenotype/cell-ontology/assets/94959119/7fa25876-ee3c-492e-b0d9-34c61403cabb">